### ISSUE TITLE
Review Nov15

### ISSUE THREAD

**Issue by YIhuiZhu** [cabinetofcuriosity/PBDB_explore#1]
1.links of dinasour names
2. plan to the museum.
3. FInd a way to visualize through time not on the website.

**Comment by iamciera** [comment]
- [ ] Add a README.md file that describes your repo.  Make sure you include a brief introduction about your project and clearly explain the files. Here is an example of a README.md that does a good job: [https://github.com/DiscoveryDNA/TFBS_presence](https://github.com/DiscoveryDNA/TFBS_presence).  
- [ ] add introduction to the paleobiology database. 
- [ ] Please read through the comments I made within the [R notebook specific](https://github.com/cabinetofcuriosity/PBDB_explore/blob/master/notebooks/PBDB%20research.ipynb), mostly you will have to just add more text in plain english what you are doing.
